recipes/cppunit/all/conanfile.py

```diff
@@ -0,0 +1,108 @@
+from conans import AutoToolsBuildEnvironment, ConanFile, tools
+from contextlib import contextmanager
+import os
+
+
+class CppunitConan(ConanFile):
+    name = "cppunit"
+    description = "UNKNOWN DESCRIPTION"
+    topics = "UNKNOWN TOPICS"
+    license = "UNKNOWN LICENSE"
+    homepage = "UNKNOWN HOMEPAGE"
```

Comment:
```suggestion
    description = "CppUnit is the C++ port of the famous JUnit framework for unit testing. Test output is in XML for automatic testing and GUI based for supervised tests."
    topics = ("conan", "cppunit", "unit-test", "tdd")
    license = " LGPL-2.1-or-later"
    homepage = "https://freedesktop.org/wiki/Software/cppunit/"
```

Reply:
Thanks, I forgot to update the template.